### PR TITLE
Rework CSS

### PR DESCRIPTION
**Pull request by misteroneill**
This reworks CSS to better match the design of the new Brightcove Player.

Also, fixes a JS bug where the player was not properly referenced.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ videojs.registerComponent('Title', Title);
 videojs.registerComponent('Shelf', Shelf);
 
 videojs.plugin('dock', function(options) {
+  let player = this;
```
Comment (gkatsev) [inline index.js]: heh, oops.

File: index.scss [hunk 2]
```diff
@@ -6,66 +6,72 @@
...
 
-  .vjs-dock-title,
-  .vjs-dock-description {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+    // Adopt some core styles from .vjs-modal-dialog from video.js
+    background: rgba(0, 0, 0, 0.8);
+    background: -webkit-linear-gradient(-90deg, rgba(0, 0, 0, 0.8) 25%, rgba(255, 255, 255, 0));
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0.8) 25%, rgba(255, 255, 255, 0));
```
Comment (gkatsev) [inline index.scss]: I just realized that this looks terrible on Safari. Do you think you'll have a chance to take a look at that?

Comment (misteroneill) [inline index.scss]: Yeah, definitely.

File: index.js (hunk 1 above)
Comment (misteroneill) [inline index.js]: :)

**Comment by gkatsev** [comment]
LGTM

**Pull request merged**